### PR TITLE
Make the task timeout configurable and derived from epoch size + add a getter

### PR DESCRIPTION
**Pull request by lenak25**

### CODE REVIEW

**Review by apalepu23 — APPROVED** [review]
looks good!

File: enigma-js/src/Enigma.js [hunk 1]
```diff
@@ -471,6 +471,38 @@ export default class Enigma {
...
+          name: 'InvalidTaskReturn',
+          message: 'Not enough time has elapsed to return task funds',
+        });
+        return;
+      }
+      try {
+        const receipt = await this.enigmaContract.methods.returnFeesForTask(task.taskId).send({
+          from: task.sender,
+        });
+        task.ethStatus = 5;
```
Comment (lenak25) [inline enigma-js/src/Enigma.js]: Please use a constant

Comment (lacabra) [reply enigma-js/src/Enigma.js]: Addressed in 6dba0a7

**Review by lacabra — APPROVED** [review]
Looks good to me

**Pull request merged**